src/k8s-engine/wsl.ts

```diff
@@ -270,6 +270,32 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
   /** The current container engine; changing this requires a full restart. */
   #currentContainerEngine = ContainerEngine.NONE;
 
+  /** Used for giving better error messages on failure to start or stop
+   * The actual underlying lima command
+   */
+  #lastCommand = '';
+
+  get lastCommand() {
+    return this.#lastCommand;
+  }
+
+  set lastCommand(value: string) {
+    console.log(`Running command ${ value }...`);
```

Comment:
Should this be using `console.debug` instead?

Reply:
No, this covers what you said a month ago: "What Jan said — we should ensure that we log things into the log files, because the error dialogs are transient."